daprdocs/content/en/developing-applications/building-blocks/state-management/query-state-store-api.md

```diff
@@ -0,0 +1,265 @@
+---
+type: docs
+title: "State Query API"
+linkTitle: "State Query API"
+weight: 600
+description: "API for querying state stores"
+---
+
+## Introduction
+
+The query API provides unified way of querying state stores regardless of underlying databases.
+
+{{% alert title="In preview" color="warning" %}}
+The state store query API is currently in [preview]({{< ref preview-features.md >}}). To enable, opt-in to `State.Query` feature.
+{{% /alert %}}
+{{% alert title="Note" color="primary" %}}
+The state store query API is implemented per component. Currently we support `MongoDB` and `Azure CosmosDB`. Support for more components is underway.
```

Comment:
Remove the word "we". 

For the list of supported state stores update these tables https://docs.dapr.io/reference/components-reference/supported-state-stores/ adding a new column. That way you do not have to mentioned them here. Put a link to these tables from here for discoverability.  So have the following text

"You can find the list of state stores that support the query API [here]({{< ref supported-state-stores.md >}})